CHANGES.rst

```diff
@@ -4,7 +4,10 @@ CHANGES
 0.7.1 (unreleased)
 ==================
 
-- Python 3 compatibility: use the implementer decorator
+- Provide IRelationBrokenEvent to be able to distinguish the event when using IObjectModifiedEvent
+  [vangheem]
+
+ - Python 3 compatibility: use the implementer decorator
```

Comment:
You've accidentally indented this line, which will likely break ReStructuredText formatting.